CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Smithy Gradle Plugin Changelog
 
+## 1.0.1 (2024-07-16)
```

Comment:
We're adding new functionality, this should be a minor version bump, not a patch.